app/request/show/model.js

```diff
@@ -0,0 +1,13 @@
+import Model, { attr, belongsTo } from '@ember-data/model';
+
+export default class RequestModel extends Model {
+  @attr('string') id;
+  @attr('string') type;
+  @attr('string') responder;
+  @attr('string') requestor;
```

Comment:
Duplicate `requestor` attribute and bad type, just remove this line, lower one is correct 👌